### ISSUE TITLE
Website - starting from scratch

### ISSUE THREAD

**Issue by bonniekim** [WebDevDivas/home#7]
Figure out how to point the outward facing website to the "master" branch (instead of gh-pages) so we can "build from scratch" instead of using boilerplate automated website.

Option 1 - manually transfer "master" branch files onto "gh-pages" branch. Delete "gh-pages" files.

Option 2 - "point" the website to the "master" branch and leave "gh-pages" in its own branch, but inactive. Keep around for reference.

I'm thinking that Option 2 is the better option... but we have to figure out if it's possible, and how.

**Comment by bonniekim** [comment]
Excerpt from our current website:

> **Creating pages manually**
> If you prefer to not use the automatic generator, push a branch named **gh-pages** to your repository to create a page manually. In addition to supporting regular HTML content, GitHub Pages support Jekyll, a simple, blog aware static site generator. Jekyll makes it easy to create site-wide headers and footers without having to copy them across every page. It also offers intelligent blog support and other advanced templating features.

Seems like it is **the gh-pages branch itself** that is linked to the website. Not sure why one would want to keep a **gh-pages** branch distinct from the **master** branch, but there it is.

It would be simple to create a duplicate branch, i.e. **resources** branch, and transfer all the **gh-pages** files into the new branch.

The question is... once the **gh-pages** branch is freed up, do we want to work _exclusively_ off of it?? Or do we want to work off of the _**master**_ branch?? What's the best practice???

If the **gh-pages** branch is designed to be the display branch (the branch off of which the website displays), what is the purpose of the master branch??

**Comment by bonniekim** [comment]
You can switch to the **gh-pages** branch using the GitHub Desktop interface.
![image](https://cloud.githubusercontent.com/assets/6983625/13908261/5351495c-eebc-11e5-9351-38ce6a2c640b.png)

Then you find the **gh-pages** branch repository in Windows Explorer/Finder through GitHub Desktop.
![image](https://cloud.githubusercontent.com/assets/6983625/13908277/aaa566c0-eebc-11e5-907a-9f4671d6518f.png)

It's curious -- switching the branch using GitHub Desktop seems to toggle "visibility" in Windows Explorer. The files can be found at the _exact same address_, but depending on which branch you have "activated" in GitHub Desktop, different files are visible...?? No idea how they're doing that.

**master** branch activated
![image](https://cloud.githubusercontent.com/assets/6983625/13908310/2c4bcc14-eebd-11e5-9b6f-83704aa8f1c8.png)

**gh-pages** branch activated
![image](https://cloud.githubusercontent.com/assets/6983625/13908318/38b3081e-eebd-11e5-87c8-330d334d5c1b.png)

**Comment by bonniekim** [comment]
Boyfriend was fiddling around with the site and somehow (no idea how) managed to get the website address to point to the master branch.

Since all our desired content already lives on the master branch, we deleted gh-pages, which no longer added functional value.

I propose that our project consists of three branches: master, bonnie-wip, madeleine-wip.

This way, we can upload and experiment onto our respective sandbox branches while preserving the master branch. Then, we can each submit pull requests to merge our sandbox branches with the master branch, and discuss in detail the extent of the changes we made. Once we decide to merge, the sandbox branch and master branch will become synced and identical. 

From there, the master branch will remain static (so that our website doesn't get messed up while we're experimenting), while we can use our sandbox however we want to, for uploading content OR for building site features.

My question is - if you merge your sandbox with the master branch, and then I, using the OLD master branch, want to merge my NEW changes with the CURRENT master branch -- will Github be smart enough to know NOT to override the current changes with old content, yet push whatever new content I'm producing?

I think it should be smart enough -- but I'm not sure about the specific mechanics of how it works.

**Comment by madeleineadkins** [comment]
Yes, I agree re sandboxes. I'm feeling too new to github to be sure how best to technically create that.Good question. I'm not sure that it would be smart enough--it compares, and I think it would just override the current with the proposed one. But maybe we can find out the answer online somewhere, or via someone we know. Meanwhile, let's proceed carefully. If we can't find out via research, then we can conduct a careful experiment, with backup files, just in case.

**Comment by madeleineadkins** [comment]
And thanks to the BF. I wish we knew what the magic trick was!
